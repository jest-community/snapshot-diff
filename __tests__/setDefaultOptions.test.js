// @flow

const snapshotDiff = require('../src/index');

const a = `
    some
    some
    some
    some
    some
    some
    some
    not
    very
    long
    script
`;
const b = `
    some
    some
    some
    some
    some
    some
    some
    not
    so
    very
    long
    script
`;

test('uses default options', () => {
  expect(snapshotDiff(a, b)).toMatchSnapshot();
});

test('uses the updated default options', () => {
  snapshotDiff.setDefaultOptions({
    colors: true,
    contextLines: 0,
    aAnnotation: 'First string',
    bAnnotation: 'Second string',
  });

  expect(snapshotDiff(a, b)).toMatchSnapshot();
});
