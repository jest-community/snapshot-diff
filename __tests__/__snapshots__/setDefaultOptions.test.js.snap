// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uses default options 1`] = `
"Snapshot Diff:
- First value
+ Second value

@@ -5,9 +5,10 @@
      some
      some
      some
      some
      not
+     so
      very
      long
      script
  "
`;

exports[`uses the updated default options 1`] = `
"Snapshot Diff:
[32m- First string[39m
[31m+ Second string[39m

[33m@@ -10,0 +10,1 @@[39m
[31m+     so[39m"
`;
